db/migrate/20220929122617_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration[7.0]
```

Comment:
- I cannot run the migrations because I get a duplication error from migration `20220914085442_add_user_ref_to_post.rb`. It is similar to that in `20220914084609_add_post_ref_to_comments.rb`